java/.ci/Jenkinsfile

```diff
@@ -919,6 +919,31 @@ pipeline {
                          }
                      }
                 }
+                stage('KAFKA TO GCS via Dstream (bytes)'){
```

Comment:
Since, existing template of KafkaToGCS already support reading json messages. Can we also have another int test case for KAFKA To GCS via Dstream (json)?

Reply:
Done